### ISSUE TITLE
Queries regarding my generated Test_info and Human_detection JSON files

### ISSUE THREAD

**Issue by Dipankar1997161** [mks0601/TF-SimpleHumanPose#79]
Hello, Thank you for posting your work! @mks0601 

I have gone through the Issues tab and tried to figure out what exactly this **human_detection.json** is.

From my perspective, human_detection is just a JSON file with keys like image_id, bbox, score, and category_id. (in short, segmented image properties)

Keys like filename, id, and image shape(H, W) are present in COCO **image_info_test_dev.json file** and **image_info_test.json file** but no score or BBOX nor keypoints
Is this correct or anything you wish to add?

**Also, it is mentioned in the README.md that human_detection.json is used during testing stages, then what is the use of the COCO test JSON file and images which is under the test and annotation folders respectively??**

I am asking this so I can properly generate those files for my custom dataset.
Because, I have my own validation set, Train set and Test set ready as per coco. 

The only confusion that arises is how your human_detection.json looks! Is it as per the details I mentioned? 
If possible, could you just share a snippet of the human_detection.json file that YOU generated? Visualizing that I can generate mine!!

**I share a snippet of my human_detection.json file and image info . Please correct me here**
1. Human_detection.json
![human_detection_json](https://user-images.githubusercontent.com/85514219/166142117-4a13907f-0957-4ac1-bdc6-677ab2cf72a2.png)

2. Image_info_test_dev.json

![image_info_test_dev 2017](https://user-images.githubusercontent.com/85514219/166142133-f2e76cb4-82da-449e-af6c-2619465ee1b4.png)

**Comment by mks0601** [comment]
human_detection.json alone is not enough as file_name is not provided in that file. The human_detection.json is not made by myself, but just an output of existing human detectin networks. human_detection.json has the required format for the COCO object detection evaluation. You can visit COCO dataset homepage.

**Comment by Dipankar1997161** [comment]
Actually, I have a JSON file (person_keypoints_scenario) that includes keys like file name, image shape, ID, bbox, and also key points. From this file, I omitted the file name and the keypoints to get the human_detection.json

But you are saying that's not enough as we do need a "file_name!"

Also, human_detection.json is used for testing after my model is created! 

PLease note:
My university has asked me to test my model on the above mentioned JSON file (person_keypoints)

SO if I just replace the human_detection with the person_keypoint json file and make the change in the config.py 
Won't that work??
